internal/cmd/root.go

```diff
@@ -9,8 +9,10 @@ import (
 )
 
 var rootCmd = &cobra.Command{
-	Use:   "rdm",
+	Use:   "remote-development-manager",
```

Comment:
```suggestion
	Use:   "rdm",
```

The binaries I share are generally named `rdm` which I don't think conflicts with other tools. Is there any reason we shouldn't use the short version here?

Reply:
Good point. 

What got me here is when you run `rdm` before installing the binary which probably doesn't make sense if you're seeing the help:

```
 $rdm
The program 'rdm' is currently not installed. You can install it by typing:
  brew install rtags
```